examples/enclave_manager/setup.py

```diff
@@ -33,7 +33,7 @@
 module_path = 'tc/sgx/trusted_worker_manager/enclave_wrapper'
 module_src_path = os.path.join(tcf_root_dir, module_path)
 
-sgx_mode_env = os.environ.get('SGX_MODE', None)
+sgx_mode_env = os.environ.get('SGX_MODE', 'SIM')
```

Comment:
Could you please add similar changes in setup.py in below path ?
examples/common/python/setup.py

Reply:
I already do that.  See the change above in `examples/common/python/setup.py`:
```
- sgx_mode_env = os.environ.get('SGX_MODE', None)
+ sgx_mode_env = os.environ.get('SGX_MODE', 'SIM')
```